pages/management/commands/create_home_page.py

```diff
@@ -0,0 +1,60 @@
+import json
+from django.core.management.base import BaseCommand
+from pages.models import HomePage
+
+class Command(BaseCommand):
+    help="create initial home page after a deployment"
+
+    def handle(self, *args, **options):
+        home_page = HomePage(
```

Comment:
Since this is editing an existing page (the page still exists, you only edited the fields), you'll need to get it and _edit_ the content in this script, instead of creating a new one. 

Get homepage (this works well, notice it's being imported at the top):
https://github.com/openstax/openstax-cms/blob/master/pages/management/commands/create_impact_page.py#L9

Then instead of `home_page = HomePage(...` you'll just edit the fields like `homepage.banner_headline='...'`. Make sure to `homepage.save()` at the end (_might_ need to come before the revision is created - not sure). 
You can also test this locally, since all your home page fields are optional, you can remove the content and make sure it populates with this.

Reply:
Script is working now